framework/src/auxkernels/MaterialStdVectorAux.C

```diff
@@ -34,5 +37,8 @@ MaterialStdVectorAux::~MaterialStdVectorAux()
 Real
 MaterialStdVectorAux::getRealValue()
 {
+  mooseAssert(_special_qp < _q_point.size(), "MaterialStdVectorAux.  special_qp specified as " << _special_qp << " but there are only " << _q_point.size() << " quadpoints in the element");
```

Comment:
Did you use `std::numeric_limits<Real>::quiet_NaN()`? Well I didn't think of exodiff. But I wouldn't let that program decide what the right thing to do is. I'd rather not test this and/or fix exodiff to add a switch that ignores NaN (or enables a true NaN == NaN comparison). 

Does anyone else have an opinion on this?